app/src/main/java/org/mifos/mobile/ui/login/LoginActivity.kt

```diff
@@ -26,28 +17,33 @@ import org.mifos.mobile.core.common.Network
 @AndroidEntryPoint
 class LoginActivity : BaseActivity() {
 
-    private val viewModel: LoginViewModel by viewModels()
-
-    private lateinit var usernameContent: String
-    private lateinit var passwordContent: String
+//    private val viewModel: LoginViewModel by viewModels()
+//    private lateinit var usernameContent: String
+//    private lateinit var passwordContent: String
```

Comment:
can we get rid of the unused code/comments in this file !!!?

Reply:
check the whole `LoginActivity` file. remove unused codes instead of commenting out...ig u forgot to remove 'em